### DISCUSSION TITLE
JSX & TypeScript

### DISCUSSION THREAD

**Discussion by KATT** [suren-atoyan/monaco-react#312] [Q&A]
Hello!

I'm trying to create an interactive playground for my project tRPC [here](https://kitchen-sink.trpc.io/react-hook-form?file=feature%2Freact-hook-form%2Findex.tsx#content) - currently just showing static code.

To make this work I need to be able to dynamically show two files that are interconnected in an editor.

I saw #56 and played around a bit with the hints I saw there, but the project seem to have changed quite a bit since.

**So the questions are**

- How do I open an editor with two files, one `.tsx` + one `.tsx`?
- Can I add a virtual file system and add TypeScript definitions for the `import`ed modules?

**Comment by suren-atoyan** [comment]
Hey @KATT 👋  could you please try [multi-model-editor](https://github.com/suren-atoyan/monaco-react#multi-model-editor)?
